leda/components/Validation/predefinedValidators.ts

```diff
@@ -2,13 +2,28 @@ import { validate } from '../../validators';
 import { NormalizedValidatorObject, PredefinedValidator } from './types';
 
 export const PREDEFINED_VALIDATORS: Record<PredefinedValidator, NormalizedValidatorObject> = {
-  cadastralNumber: {
-    validator: validate.cadastralNumber,
-    invalidMessage: 'Введите кадастровый номер',
+  // Common validators
+  creditCardNumber: {
+    validator: validate.creditCardNumber,
+    invalidMessage: 'Card number is not valid'
   },
   email: {
     validator: validate.email,
-    invalidMessage: 'Введите e-mail',
+    invalidMessage: 'E-mail is not valid',
+  },
+  password: {
+    validator: validate.password,
+    invalidMessage: 'Password should have upper and lowercase latin letters and numbers, 8 symbols and more',
+  },
+  url: {
+    validator: validate.url,
+    invalidMessage: 'Url is not valid',
+  },
+
+  // Russia specific validators  
+  cadastralNumber: {
```

Comment:
Maybe we should drop this off? It doesn't look really generic. I can't see no reason why we should support it out of the box

Reply:
There is an issue about it on the board, so we'll do it